roles/openshift_node/tasks/bootstrap.yml

```diff
@@ -42,6 +45,31 @@
     path: /etc/origin/.config_managed
   register: rpmgenerated_config
 
+- name: create directories for bootstrapping
+  file:
+    state: directory
+    dest: "{{ item }}"
+  with_items:
+  - /root/openshift_bootstrap
+  - /var/lib/origin/openshift.local.config
+  - /var/lib/origin/openshift.local.config/node
```

Comment:
The root/openshift_bootstrap is where we can place the variable file which tells the server which group type he is.  We can pass other data non related to openshift in this file so I'd prefer not to be in the openshift directory path.  The bootstrap.yml playbook gets laid down inside of this /root/openshift_bootstrap.  If you'd prefer that it move then that's fine.  It can do more than bootstrap openshift as it is also fixing dns.